docs/code-snippets/excel-snippets.yaml

```
@@ -361,397 +229,249 @@ Excel.ChartCollection#add:member(1):
+    await Excel.run(async (context) => { 
+        var seriesCollection = context.workbook.worksheets.getItem("Sheet1").charts.getItem("Chart1").series;
         seriesCollection.load('items');
-        return ctx.sync().then(function() {
-            console.log(seriesCollection.items[0].name);
-        });
-    }).catch(function(error) {
-        console.log("Error: " + error);
-        if (error instanceof OfficeExtension.Error) {
-            console.log("Debug info: " + JSON.stringify(error.debugInfo));
-        }
+        await context.sync();
+        
+        console.log(seriesCollection.items[0].name);
     });
 Excel.ChartSeriesCollection#load:member(2):
-  - |-
-    // Getting the names of series in the series collection.
-    Excel.run(function (ctx) { 
-        var seriesCollection = ctx.workbook.worksheets.getItem("Sheet1").charts.getItem("Chart1").series;
-        seriesCollection.load('items');
-        return ctx.sync().then(function() {
-            for (var i = 0; i < seriesCollection.items.length; i++)
-            {
-                console.log(seriesCollection.items[i].name);
-            }
-        });
-    }).catch(function(error) {
-        console.log("Error: " + error);
-        if (error instanceof OfficeExtension.Error) {
-            console.log("Debug info: " + JSON.stringify(error.debugInfo));
-        }
-    });
   - |-
     // Get the number of chart series in collection.
-    Excel.run(function (ctx) { 
-        var seriesCollection = ctx.workbook.worksheets.getItem("Sheet1").charts.getItem("Chart1").series;
+    await Excel.run(async (context) => { 
+        var seriesCollection = context.workbook.worksheets.getItem("Sheet1").charts.getItem("Chart1").series;
         seriesCollection.load('count');
-        return ctx.sync().then(function() {
-            console.log("series: Count= " + seriesCollection.count);
-        });
-    }).catch(function(error) {
-        console.log("Error: " + error);
-        if (error instanceof OfficeExtension.Error) {
-            console.log("Debug info: " + JSON.stringify(error.debugInfo));
-        }
+        await context.sync();
+
+        console.log("series: Count= " + seriesCollection.count);
     });
 Excel.ChartTitle#load:member(2):
-  - |-
-    // Get the text of Chart Title from Chart1.
-    Excel.run(function (ctx) { 
-        var chart = ctx.workbook.worksheets.getItem("Sheet1").charts.getItem("Chart1");    
-        
-        var title = chart.title;
-        title.load('text');
-        return ctx.sync().then(function() {
-                console.log(title.text);
-        }).catch(function(error) {
-            console.log("Error: " + error);
-            if (error instanceof OfficeExtension.Error) {
-                console.log("Debug info: " + JSON.stringify(error.debugInfo));
-            }
-        });
-    });
   - |-
     // Set the text of Chart Title to "My Chart" and Make it show on top of the chart without overlaying.
```

If overlay=true then it seems like this comment is wrong. 
```suggestion
    // Set the text of the chart title to "My Chart" and display it as an overlay on the chart.
```